codegenerator/cli/npm/package.json.tmpl

```diff
@@ -1,7 +1,7 @@
 {
   "name": "${node_pkg}",
   "version": "${version}",
-  "description": "A ️latency and sync speed optimized, developer friendly blockchain data indexer.",
+  "description": "A latency and sync speed optimized, developer friendly blockchain data indexer.",
```

Comment:
There was an unsupported unicode char for space

Reply:
Wow, wonder how that happened 😆